flink-core/src/main/java/org/apache/flink/util/FileUtils.java

```diff
@@ -315,7 +315,11 @@ private static void deleteFileOrDirectoryInternal(File file) throws IOException
 	}
 
 	private static void deleteDirectoryInternal(File directory) throws IOException {
-		if (directory.isDirectory()) {
+		if (Files.isSymbolicLink(directory.toPath())) {
+			//noinspection ResultOfMethodCallIgnored
```

Comment:
Please remove `//noinspection ResultOfMethodCallIgnored`. Even though most of us have IntelliJ, it's still noise for anyone how does not have an active warning here.